### ISSUE TITLE
installation fails on community edition

### ISSUE THREAD

**Issue by QuentinAmbard** [databricks-demos/dbdemos#8]

**Comment by ottok92** [comment]
```
API CALL ERROR - can't read json. status: 403 RBAC: access denied - URL: https://community.cloud.databricks.com/api/2.0/clusters/create - Expecting value: line 1 column 1 (char 0)
```

**Comment by QuentinAmbard** [comment]
Sorry Ottok, we still don't support community edition as we're missing some features here :/ 
will work on a fix & in the immediate term a better error message

**Comment by rahulwankhade** [comment]
Dear Databricks team,

Request you enable the Demos on Community edition so that we can understand & learn more about the platform.

**Comment by QuentinAmbard** [comment]
hi @rahulwankhade , we released dbdemos 0.3 and it should now work in the community edition.
It'll skip the dashboard + DLT installation but load the notebooks
Give it a try and let le know
I'm closing the issue, but please re-open it if it's not working as expected for you!
